apps/prairielearn/src/lib/question-render.js

```diff
@@ -557,8 +535,9 @@ export async function getAndRenderVariant(variant_id, variant_seed, locals) {
 
 /**
  * @param {import('./db-types.js').GradingJob | null} job
+ * @returns {import('../components/SubmissionPanel.html.js').GradingJobStats | null}
  */
-function buildGradingJobStats(job) {
+export function buildGradingJobStats(job) {
```

Comment:
I am debating maybe moving this function to the submission panel component code, since now it's only used there. Thoughts?

Reply:
I had to move it anyway because of a circular dependency. Done.